### PR TITLE
Add commitSummaries field to MergeConfig model

### LINKED ISSUE #364: Add commitSummaries field to MergeConfig model
`MergeConfig` model lacks `commitSummaries` field

https://developer.atlassian.com/server/bitbucket/rest/v806/api-group-repository/#api-api-latest-projects-projectkey-repos-repositoryslug-settings-pull-requests-post

### Expected Behavior
`MergeConfig` model has `commitSummaries` field, and it is possible to update `Commit summaries` option on `Merge strategies` tab.

### Current Behavior
Updating of `Commit summaries` option is not possible. Each request to this API resets the value to default 20

### Context
I need to be able to update `Commit summaries` option

### Your Environment

### PR DESCRIPTION
**Pull request by neutrino1911**
closes #364

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java'
+    id 'java-library'
```
Comment (cdancy) [inline build.gradle:3]: Can we remove this change?

**Comment by cdancy** [comment]
@neutrino1911 aside from 1 minor comment everything looks good.

File: build.gradle (hunk 1 above)
Comment (neutrino1911) [reply build.gradle:3]: sure, but build fails w/o it

**Comment by neutrino1911** [comment]
@cdancy what is next? I did what you asked for

**Pull request merged**